File: src/001/java/001.java
public class multi {
  private final int target = 999;

  public static void main(String... args) {
    multi m = new multi();

    System.out.println(m.sumDiv(3) + m.sumDiv(5) - m.sumDiv(15));
  }

  private double sumDiv(int n) {
    final double p = Math.floor(this.target / n);
    return  n * (p * (p + 1)) / 2;
  }
}

File: src/001/clojure/001.clj
(def target 999)
(defn- sumDivisibleBy [n]
  (let [p (Math/floor (/ target n))]
    (/ (* n (* p (+ 1 p))) 2) ))

(print (- (+ (sumDivisibleBy 3) (sumDivisibleBy 5)) (sumDivisibleBy 15)))
